src/lib/consensus/proof_of_stake.ml

```diff
@@ -2734,17 +2735,36 @@ module Hooks = struct
      and type protocol_state_var := Protocol_state.var
      and type snark_transition_var := Snark_transition.var = struct
     (* TODO: only track total currency from accounts > 1% of the currency using transactions *)
+
+    let error ~module_ ~location ~logger fmt =
```

Comment:
I don't really understand this helper since it's basically the same length as the expression it maps to.

Reply:
Yeah -- it was doing more before. Removed now.